allauth/account/forms.py

```diff
@@ -432,6 +432,8 @@ def clean_email(self):
                               " with this account."),
             "different_account": _("This e-mail address is already associated"
                                    " with another account."),
+            'over_account_limit': _("You cannot add more email addresses as "
+                                    "you are at maximum allowance.")
```

Comment:
Would be nice if this message simply said: "You cannot setup more than %d e-mail addresses.".